test/plugin/constants.rb

```diff
@@ -748,6 +748,7 @@ module Constants
     'responseSize' => 65,
     'userAgent' => 'USER AGENT 1.0',
     'remoteIp' => '55.55.55.55',
+    'serverIp' => '66.66.66.66',
```

Comment:
It feels like you were planning to add a test. Is the file missing from this branch?

Reply:
The tests iterate through all fields defined in these maps (so you don't have to hard-code a new test anytime a new field is added), so this effectively does add tests.

Reply:
Oh, I see. Thanks!

Reply:
Yup, exactly what @jkschulz said! Isn't that great? 😃

Reply:
I was going to stay silent, but since you asked... unfortunately, I don't think so. Data-driven tests are tricky. If the test gets broken and doesn't test some or all of the fields, you'll never know, meaning your coverage will drop silently.